src/react/databrowser/objects/ObjectListTable.tsx

```diff
@@ -50,30 +97,20 @@ export default function ObjectListTable({
   prefixWithSlash,
 }: Props) {
   const { accountName } = useParams<{ accountName: string }>();
-
-  const [isTableScrollbarVisible, setIsTableScrollbarVisible] = useState(false);
-  const [tableWidth, setTableWidth] = useState(0);
+  const { pathname } = useLocation();
+  const query = useQueryParams();
+  const [isTableScrollbarVisible] = useState(false);
+  const [tableWidth] = useState(0);
   const dispatch = useDispatch();
-  const nextMarker = useSelector(
-    (state: AppState) => state.s3.listObjectsResults.nextMarker,
-  );
-  const loading = useSelector(
-    (state: AppState) => state.networkActivity.counter > 0,
-  );
+  const { bucketName: bucketNameParam } = useParams();
```

Comment:
I guess we could improve the typing by the way.
```suggestion
 const { bucketName: bucketNameParam } = useParams<{ bucketName: string }>();
```